dotcom-rendering/src/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -303,15 +303,19 @@ type RemoteBannerProps = BannerProps & {
 };
 
 const RemoteBanner = ({ module, fetchEmail }: RemoteBannerProps) => {
-	const [Banner, setBanner] = useState<React.ElementType>();
+	const [Banner, setBanner] = useState<React.ElementType | null>(null);
 
 	useEffect(() => {
 		setAutomat();
 
-		window
-			.guardianPolyfilledImport(module.url)
-			.then((bannerModule: { [key: string]: JSX.Element }) => {
-				setBanner(() => bannerModule[module.name] ?? null); // useState requires functions to be wrapped
+		(module.name === 'SignInPromptBanner'
```

Comment:
This is the change to fetch the DCR version of the banner components (using webpack chunks) instead of fetching the SDC version of the components